client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +87,198 @@ function HelpModal({
   challengeBlock,
   challengeTitle
 }: HelpModalProps): JSX.Element {
+  const [showHelpForm, setShowHelpForm] = useState(false);
+  const [description, setDescription] = useState('');
+  const [readSearchCheckbox, setReadSearchCheckbox] = useState(false);
+  const [similarQuestionsCheckbox, setSimilarQuestionsCheckbox] =
+    useState(false);
+  const DESCRIPTION_MIN = 50;
+  const DESCRIPTION_MAX = 500;
```

Comment:
```suggestion
  const DESCRIPTION_MAX = 500;
  const formRef = useRef<HTMLFormElement>(null);

  useEffect(() => {
    if (showHelpForm) {
      formRef.current?.querySelector('input')?.focus();
    }
  }, [showHelpForm]);
```

Currently, when the "Create a help post on the forum" button is clicked, focus is lost when the second dialog opens and thus screen reader users hear nothing. We must manually set the focus so that they get an indication that something has changed. I think the most logical place to put the focus is on the first checkbox.

If we continue to have the Back button on the second dialog then we will also need to manually set focus when they go back to the first dialog. But I'm not adding that logic now because I'm hoping we reconsider having the Back button.